.gitignore

```diff
@@ -8,3 +8,4 @@
 /demo/sovereign/target
 /demo/sovereign/demo-rollup/demo_data
 /target
+/xtask/*.log
```

Comment:
I don't think it's a good idea to mix logs with the source files.

Recall my suggestion where we could store all the test data files (for zombienet, sovereign demo rollup, and now logs) in a single place. Do you disagree with it?

Reply:
Absolutely not! I would really like to have all data in the same place but to be fair, with the current setup, I would not know how to place the zombienet folder, the demo-data folder, and the logs in the same location.

The problem is about the zombienet and logs, but mainly the sovereign rollup as described here: https://github.com/thrumdev/blobs/issues/227